rolebinding-sre-namespace.yaml

```diff
@@ -0,0 +1,13 @@
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: sre-user-access
+  namespace: sre
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: sre-user-full-access
+subjects:
+- kind: Group
+  namespace: sre
+  name: 785e6c8f-6e5f-4543-a88d-fa391591ea61
```

Comment:
These create the rolebindings that can be used to limit access via RBAC. Groups must be created with azure cli or some other way.